### ISSUE TITLE
Error loading  Python DLL

### ISSUE THREAD

**Issue by AyumuUwU** [BEEmod/BEE2.4#1801]
### BEE2 Application Version

BEE2.4

### BEE2 Package Version

64xbit

### What operating system are you running on?

Windows 10 (64bit)

### Description of the bug

i cant run it it says  Error loading Phython DLL 
C:\Users \86x\Steam\Dropbox\portal2

### Expected behavior

i need help to fix that

### Steps to reproduce

i have phython 3.8.0 installed

### Logs and Reports

```shell
pls help
```

**Comment by Luke18033** [comment]
Please copy the actual error message, that is definitely not the correct path...

**Comment by AyumuUwU** [comment]
the left button u know + C

**Comment by Luke18033** [comment]
Just attach a screenshot then

**Comment by AyumuUwU** [comment]
![screen shot error](https://user-images.githubusercontent.com/98408543/151007045-d522d1bc-25d5-40e3-aba1-64c20bf7ede3.PNG)

**Comment by AyumuUwU** [comment]
i blured that numbers because i dont want it to share u know

**Comment by AyumuUwU** [comment]
its german so ill translate:
The specified module was not found

**Comment by AyumuUwU** [comment]
helo u there

**Comment by Luke18033** [comment]
Yes, please be patient... You need to extract the contents of the ZIP, didn't Windows warn you about that?

**Comment by AyumuUwU** [comment]
i did extract it

**Comment by AyumuUwU** [comment]
everything

**Comment by AyumuUwU** [comment]
ok im closing it the support is soo bad

**Comment by Luke18033** [comment]
Please calm down, do you realize we're people with actual lives as well who don't have time to immediately respond to every issue?

**Comment by FrozenDragon0** [comment]
So just insult a Collaborator over not getting a response in time.... Seriously learn to get some patience, not everyone can respond immediately.
